### PR TITLE
Add "--needed" to Arch command to avoid reinstalling packages

### PR DESCRIPTION
**Pull request by Sovenok-Hacker**
Tiny fix for doc/compiling/linux.md, do not reinstall arch packages if installed

### CODE REVIEW

**Comment by appgurueu** [comment]
I'm not familiar with Arch, but extrapolating from Debian: Doesn't "reinstalling" just update packages (or do nothing if the packages are already up to date)?

**Comment by sfan5** [comment]
No, Arch actually downloads and installs packages again when you don't use that flag.

**Comment by Sovenok-Hacker** [comment]
> No, Arch actually downloads and installs packages again when you don't use that flag.

As Manjaro user, I can prove that`s truth

File: doc/compiling/linux.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ For openSUSE users:
 
 For Arch users:
 
-    sudo pacman -S base-devel libcurl-gnutls cmake libxi libpng sqlite libogg libvorbis openal freetype2 jsoncpp gmp luajit leveldb ncurses zstd gettext
+    sudo pacman -S base-devel libcurl-gnutls cmake libxi libpng sqlite libogg libvorbis openal freetype2 jsoncpp gmp luajit leveldb ncurses zstd gettext --needed
```
Comment (sfan5) [inline doc/compiling/linux.md:33]:
```suggestion
    sudo pacman -S --needed base-devel libcurl-gnutls cmake libxi libpng sqlite libogg libvorbis openal freetype2 jsoncpp gmp luajit leveldb ncurses zstd gettext
```

**Pull request merged**